assets/js/pages/Auth.tsx

```diff
@@ -1,25 +1,26 @@
 import React, { useEffect } from "react";
 import { useNavigate } from "react-router-dom";
 
+import axios from "../services/index";
+
 const Auth = () => {
   const navigate = useNavigate();
   const redirectToHomePage = () => navigate("/");
 
-  const fetchToken = (google_response) => {
-    const request: RequestInit = {
-      method: "POST",
-      headers: { "Content-Type": "application/json" },
-      body: JSON.stringify(google_response),
-    };
-
-    fetch(window.location.origin + "/auth", request)
-      .then((resp) => resp.json())
-      .then((data) => localStorage.setItem("jwt", data["token"]));
+  const getToken = async (google_response) => {
```

Comment:
Shouldn't function getToken, return token?

Reply:
Right. I've changed it to `fetchToken`